B33blebroxx/INDIVIDUAL-ASSIGNMENT-Team-Roster
# Got team details rendering team members, though deleting team still d…

…eletes players from that team

## Description
Got team detail page showing team members.

## Related Issue
Team Stretch
## Motivation and Context


## How Can This Be Tested?
Go into player page and add a player, once created go to the team you put the player on and you should see your new player on the screen.

## Types of changes

- [x ] New feature (non-breaking change which adds functionality)
